index.md

```diff
@@ -64,25 +67,6 @@ see the issue for details on the typos fixed
 
 fixes issue #12
 ```
-
-## Introduction
-
-In software development, it's been my experience that bugs are most often introduced at the boundaries between applications.
```

Comment:
for now let's add this section back and get the new website over the finish line. I'd then be comfortable with talking about moving this section into another part of the website.

I don't want to trim this down to one sentence, I would prefer that we have a full page that speaks to why using conventional commits is an awesome idea -- with the spec itself centering around what this using this convention actual entails.

Reply:
As I told you before, there's already a section in the website.
We can use that one without adding a new layout that someone needs to write (an internal page have to be different from the homepage).

Reply:
having reflected on this, and looking back at the structure of semver.org, I continue to come back to the argument:

_there's no good reason for us to remove the Introduction section._

I'm happy to work with you and, @zeke , and @hbetts to rework the introductory sentence. And in another pull request I'd be happy to word-smith the content of the Introduction paragraph. 
But, given that the structure of the "Conventional Commits" document is based off the structure of the "Semver" document, and that the two RFCs are closely related, I see a lot of value in keeping close to the same structure to the document.

Removing the Introduction section:

* removes a compelling description of why this specification exists.
* deviates from the structure of the document on semver.org.

tldr; I continue to be a strong -1 on this pull request. But, would love for us to reach a resolution so that we can land the new website design with content we all agree on.

Reply:
As I told sever times when you read the spec is redundant and doesn't fit the section, we have a specified section to discuss why to use the spec.

Let's move forward, I'll remove it from this PR and discuss in a different issue so we can close this one.

Reply:
sounds like we've agreed to disagree for now 😝

let's get a few third parties discussing this in another issue, since we seem to have trouble agreeing on this one point.

I still think it's worthwhile to rewrite the first sentence though? I'm pretty into just making it jump immediately into the functionality of the spec.